Gemfile.lock

```diff
@@ -448,12 +445,12 @@ GEM
     simplecov-html (0.12.3)
     simpleidn (0.2.1)
       unf (~> 0.1.4)
-    sinatra (2.2.2)
-      mustermann (~> 2.0)
-      rack (~> 2.2)
-      rack-protection (= 2.2.2)
+    sinatra (3.0.2)
```

Comment:
The changelog looks good for this https://github.com/sinatra/sinatra/blob/8ff496bd4877520599e1479d6efead39304edceb/CHANGELOG.md#300--2022-09-26

I wouldn't be against pinning to 2.x for a bit longer until 3.1 is out to reduce potential risk

Reply:
Not a blocker for me: https://github.com/rapid7/metasploit-framework/pull/17167#issuecomment-1293551343